uitest/Dialogs/AddWidgetDialog.cs

```diff
@@ -0,0 +1,90 @@
+﻿// Copyright (c) Microsoft Corporation and Contributors
+// Licensed under the MIT license.
+
+using DevHome.UITest.Extensions;
+using DevHome.UITest.Pages;
+using OpenQA.Selenium.Appium;
+using OpenQA.Selenium.Appium.Windows;
+
+namespace DevHome.UITest.Dialogs;
+
+/// <summary>
+/// Dialog model class for the add widget dialog
+/// </summary>
+public class AddWidgetDialog : PageDialog<DashboardPage>
+{
+    private WindowsElement SSHNavigationItem => Driver.FindElementByAccessibilityId($"{Configuration.Widget.IdPrefix}!App!!{Configuration.Widget.Provider}!!SSH_Wallet");
```

Comment:
I imagine `Driver.FindElementByAccessibilityId()` has to traverse the whole tree to find each item. Could we point it to a known ancestor element to make the search quicker?
E.g., instead of looking in the whole Application, look only in the Dialog

Reply:
I am not sure how the framework implements the search (traverse tree everytime, lookup pre-indexed ids, ...). Today it's pretty fast to find elements from my testing, but if we notice that the test method execution time is slow we can explore the option to use an intermediate element at a higher level in the tree 👍